tests/unit/test_wheel.py

```diff
@@ -278,7 +278,7 @@ def test_std_install(self, data, tmpdir):
             scheme=self.scheme,
             req_description=str(self.req),
         )
-        self.assert_installed(0o644)
+        self.assert_installed()
```

Comment:
Nit, I would favour reverting this since the implicit `0o644` is not obvious IMO.

Reply:
@uranusjr I think it reads better with the default, since here (as well as in the new test added in this PR) we don't want to test permissions. Only in the specific test that checks for permission, we pass a specific expected_permissions.

Reply:
I’m not following. The new implementation still implicitly checks the permission against `0o644`, doesn’t it? The proposed change seems to make the fact less obviously than it was to me.

Reply:
Ok I reverted that bit.